renderer/src/hooks/StationActivity.tsx

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react'
+import { getTotalJobsCompleted, getAllActivities, getTotalEarnings } from '../lib/station-config'
+import { ActivityEventMessage } from '../typings'
+
+const useStationActivity = () : [number, number, ActivityEventMessage[]] => {
+  const [totalJobs, setTotalJobs] = useState<number>(0)
+  const [activities, setActivities] = useState<ActivityEventMessage[]>([])
+  const [totalEarnings, setTotalEarnigs] = useState<number>(0)
+
+  useEffect(() => {
+    const loadStoredInfo = async () => {
+      Promise.all([
+        (async () => { setTotalJobs(await getTotalJobsCompleted()) })(),
+        (async () => { setActivities(await getAllActivities()) })(),
+        (async () => { setTotalEarnigs(await getTotalEarnings()) })()
```

Comment:
We could write this in a less verbose way, it's killing me... I'll think about something and let you know